deploy/helm/kube-cleanup-operator/values.yaml

```diff
@@ -106,7 +106,7 @@ rbac:
 ## Arguments for kube-cleanup-operator
 ##
 args: []
-  # - --namespace=default
+  # - --namespace={{ .Release.Namespace }}
```

Comment:
Not sure if it is a good idea to change here. I think now it's different from the default, which is `default` namespace and it needs to be override at deployment time. I think this example helps when people do not want to set a override for each namespace they deploy the helm chart.

Reply:
Well, this particular line is a comment, it has zero effect apart on the render/release process. 
Also you can't use helm's template variables in values, because values are used to render templates.

Reply:
with `tpl` function inside helm chart, whatever template we put here as string will get evaluated when helm render the template. So in this case, namespace will be replaced when helm render the chart.

Reply:
And I agree it's just a comment, however sometimes i refer to comments and think it's the default, or look for examples. Maybe I'd better to change this back to default value which is `default`, and add a description instead.

Reply:
1. well values are supposed to be "values" for the rendering and should not have any meta templating. passing templates to the templates it's not something I want to ever support.
2. There are 2 modes you can run this operator: global and per-namespace. If you don't want to deal with namespaces you can deploy it at global scope, otherwise namespace name should be provided from the outside to the arg.